modules/biscuit/bsconv/main.nf

```diff
@@ -0,0 +1,39 @@
+process BISCUIT_BSCONV {
+    tag "$meta.id"
+    label 'process_long'
+
+    conda (params.enable_conda ? "bioconda::biscuit=1.0.2.20220113" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/biscuit:1.0.2.20220113--h81a5ba2_0':
+        'quay.io/biocontainers/biscuit:1.0.2.20220113--h81a5ba2_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path(index)
+
+    output:
+    tuple val(meta), path("*.bsconv.bam"), emit: bsconv_bam
```

Comment:
Should just be `*.bam` unless the tool itself writes it like so

see last bullet of this: https://nf-co.re/developers/adding_modules#naming-conventions

If you're worried about filename conflicts